src/Cake.Issues.Markdownlint/MarkdownlintIssuesAliases.MarkdownlintCli.cs

```diff
@@ -0,0 +1,125 @@
+﻿namespace Cake.Issues.Markdownlint
+{
+    using Cake.Issues.Markdownlint.MarkdownlintCli;
+    using Core;
+    using Core.Annotations;
+    using Core.IO;
+
+#pragma warning disable SA1601 // Partial elements must be documented
```

Comment:
Why was the warning disabled instead of adding documentation for this class?

Reply:
Documentation is provided through [main partial class](https://github.com/cake-contrib/Cake.Issues.Markdownlint/blob/f119dda6179ce4a474705de3427d867154bc13bf/src/Cake.Issues.Markdownlint/MarkdownlintIssuesAliases.cs#L5-L7)